client/src/components/tests/UserInfo2.test.js

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+import Enzyme, { shallow, mount } from 'enzyme';
+
+import UserInfo2 from '../UserInfo2';
+
+describe('UserInfo2', () => {
+  const props = {
+    stockInfo: [
```

Comment:
can do additional testing with some filled out stockInfo and test if it matches in the shallow render.